Tools/TestWebKitAPI/Tests/mac/FontManagerTests.mm

```diff
@@ -198,8 +198,10 @@ - (void)selectNextWord
     [webView waitForNextPresentationUpdate];
 
     auto expectSameAttributes = [](NSFont *font1, NSFont *font2) {
-        auto fontAttributes1 = font1.fontDescriptor.fontAttributes;
-        auto fontAttributes2 = font2.fontDescriptor.fontAttributes;
```

Comment:
(It's not super critical since API tests terminate between each test, but) it would be nice to avoid leaking these.